interp/constrexpr.ml

```diff
@@ -19,11 +19,17 @@ type universe_decl_expr = (lident list, Glob_term.glob_constraint list) UState.g
 type ident_decl = lident * universe_decl_expr option
 type name_decl = lname * universe_decl_expr option
 
+type notation_specificity = LastLonelyNotation | NotationInScope of string
```

Comment:
What does `specificity` mean, in this context?

Reply:
See above. I'm not particularly satisfied with the use of the word `specificity` here. In any case, thanks a lot for reviewing.